tools/python/maps_generator/utils/file.py

```diff
@@ -24,6 +25,19 @@
 logger = logging.getLogger("maps_generator")
 
 
+def is_file_uri(url: AnyStr) -> bool:
+    return urlparse(url).scheme == "file"
+
+def file_uri_to_path(url : AnyStr) -> AnyStr:
+    file_uri = urlparse(url)
+    file_path = file_uri.path
+
+    #URI is somehting like "file://~/..."
+    if file_uri.netloc == '~':
+        file_path = f'~{file_uri.path}'
+    
+    return os.path.expanduser(file_path)
```

Comment:
Should expanduser be called only if the path starts with ~?

Reply:
Sure, it doesn't need to be called in every case.